### PR TITLE
ADEN-8150 Rename mobile_top_leaderboard to top_leaderboard and add MEGA

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/99c778594a5bfe6e40fe061218b2b889d91e4a4c/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/017ccdc9a37fb22344ffc3265c02fed70032eaeb/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/30d0bd9e07d6e1739e0d6543a74e2bd12ce37630/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/5c7efb4b47d72da01dcb605d0c95980b0379c630/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsPrebidObject.java#L3](https://github.com/Wikia/selenium-tests/blob/5c7efb4b47d72da01dcb605d0c95980b0379c630/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java#L3): Remove this unused import 'com.wikia.webdriver.common.contentpatterns.AdsContent'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/854f399a94ec81e2eec63f2dbbc098d54a7d52be/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsPrebidObject.java#L3](https://github.com/Wikia/selenium-tests/blob/854f399a94ec81e2eec63f2dbbc098d54a7d52be/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java#L3): Remove this unused import 'com.wikia.webdriver.common.contentpatterns.AdsContent'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java [hunk 1]
```diff
@@ -23,13 +22,7 @@ public AdsPrebidObject(WebDriver driver, String testedPage) {
...
   public void verifyKeyValues(String slotName, String adapter, String size, String price) {
 
-    WebElement gptNode;
-
-    if (slotName.contains("mobile")) {
-      gptNode = driver.findElement(By.cssSelector("#" + slotName));
-    } else {
-      gptNode = driver.findElement(By.cssSelector("#" + slotName + " " + GPT_NODE));
-    }
+    WebElement gptNode = driver.findElement(By.cssSelector("#" + slotName));
```
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java]: It should, maybe it's good idea to verify it during QA.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/f17a4f1d22b8b5614ef3011cab57ac880fdae096/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsPrebidObject.java#L3](https://github.com/Wikia/selenium-tests/blob/f17a4f1d22b8b5614ef3011cab57ac880fdae096/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java#L3): Remove this unused import 'com.wikia.webdriver.common.contentpatterns.AdsContent'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsContent.java#L38](https://github.com/Wikia/selenium-tests/blob/602e581712b30f72b0fb8d7d198b4d7ea107e021/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java#L38): Define a constant instead of duplicating this literal "#TOP_LEADERBOARD" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsPrebidObject.java#L3](https://github.com/Wikia/selenium-tests/blob/602e581712b30f72b0fb8d7d198b4d7ea107e021/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java#L3): Remove this unused import 'com.wikia.webdriver.common.contentpatterns.AdsContent'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**